components/docs-chef-io/content/automate/ha-aws-deployment-prerequisites.md

```diff
@@ -0,0 +1,203 @@
++++
+title = "AWS Deployment Prerequisites"
+
+draft = false
+
+gh_repo = "AWS Deployment Prerequisites"
+[menu]
+  [menu.automate]
+    title = "AWS Deployment Prerequisites"
+    parent = "automate/deploy_high_availability"
+    identifier = "automate/settings/ha-aws-deployment-prerequisites.md AWS Deployment Prerequisites"
+    weight = 13
++++
+
+{{< warning >}}
+{{% automate/ha-warn %}}
+{{< /warning >}}
+
+{{< warning >}}
+The below pre-requisites are according to our organizational standard. If you are using any specified version not mentioned here or a third party extensions or software you can reach out to the CAMs and our Customer Support Team.
+{{< /warning >}}
+
+Before installing Chef automate HA on AWS deployment, ensure you have taken a quick tour of this pre-requisite page.
```

Comment:
`Before installing Chef automate HA on AWS deployment` - can you rewrite this sentence?